user/plan9port/template.py

```diff
@@ -0,0 +1,54 @@
+pkgname = "plan9port"
+pkgver = "0_git20240110"
+pkgrel = 0
+_commit = "be7c68f6954f7dcaa53403e0f600716f65a13d32"
+hostmakedepends = ["perl"]
+makedepends = [
+    "fontconfig-devel",
+    "freetype-devel",
+    "libx11-devel",
+    "libxext-devel",
+    "libxt-devel",
+    "linux-headers",
+]
+pkgdesc = "Port of programs from Plan 9"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "MIT"
+url = "https://9fans.github.io/plan9port"
+source = f"https://github.com/9fans/plan9port/archive/{_commit}.zip"
+sha256 = "931b96e814b31769bf62b2ef2189ac2f474207d079932987c468b1c86854fb39"
+options = ["!cross", "!lintstatic"]
+
+
+def do_configure(self):
+    with open(self.cwd / "LOCAL.config", "w") as config:
+        config.write(f"CC9=" + self.get_tool("CC") + "\n")
+        config.write(f"CC9FLAGS='" + self.get_cflags(shell=True) + "'\n")
```

Comment:
ah missed this before- ldflags are ignored too, so you want to put them in this same cc9flags array i guess (gives a bunch of warnings but there's no other way that i see)